src/components/v5/frame/ColonyHome/partials/FundsCards/partials/FundsCardsTotalDescription.tsx

```diff
@@ -0,0 +1,25 @@
+import { ArrowUp, ArrowDown } from '@phosphor-icons/react';
+import React from 'react';
+
+type ArrowDirection = 'up' | 'down';
+
+interface FundsCardsTotalDescriptionProps {
+  direction: ArrowDirection;
+}
+
+// @TODO create logic for this component, props etc
+export const FundsCardsTotalDescription: React.FC<
+  FundsCardsTotalDescriptionProps
+> = ({ direction }) => {
+  const Arrow = direction === 'up' ? ArrowUp : ArrowDown;
+
+  return (
+    <div className="flex w-full justify-between text-xs">
+      <span className="text-gray-400">Native 1.56M CLNY</span>
+      <span className="flex items-center text-blue-400">
```

Comment:
This should have `font-medium` (The arrow also looks a bit thicker on the figma design than here).

Reply:
I have updated it, but I am not sure that svg will get bolder :( I think this is something with imported icon or something like this